src/components/Cashout/Components/Initial.view.tsx

```diff
@@ -56,10 +56,17 @@ export const InitialCashoutView = ({
         showError: boolean
         errorMessage: string
     }>({ showError: false, errorMessage: '' })
+    // TODO: @Hugo0 value is ambigous with price - it should be tokenAmount and tokenPrice. But this means changes across a bunch of files.
     const [_tokenValue, _setTokenValue] = useState<string | undefined>(
         inputDenomination === 'TOKEN' ? tokenValue : usdValue
     )
 
+    useEffect(() => {
+        if (!_tokenValue) {
+            _setTokenValue('0')
+        }
+    }, [])
```

Comment:
intention: start off with 0 as selected amount to trigger min cashout message.
Maybe there's bettter way to do this